### PR TITLE
README: add missing verb

### PR DESCRIPTION
**Pull request by solt87**
Apparently there was a verb missing. (Feel free to re-word it if needed.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ logger.add(sys.stderr, format="{time} {level} {message}", filter="my_module", le
 
 This function should be used to register [sinks](https://loguru.readthedocs.io/en/stable/api/logger.html#sink) which are responsible for managing [log messages](https://loguru.readthedocs.io/en/stable/api/logger.html#message) contextualized with a [record dict](https://loguru.readthedocs.io/en/stable/api/logger.html#record). A sink can take many forms: a simple function, a string path, a file-like object, a coroutine function or a built-in Handler.
 
-Note that you may also  a previously added handler by using the identifier returned while adding it. This is particularly useful if you want to supersede the default `stderr` handler: just call [`logger.remove()`](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.remove) to make a fresh start.
+Note that you may also call a previously added handler by using the identifier returned while adding it. This is particularly useful if you want to supersede the default `stderr` handler: just call [`logger.remove()`](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.remove) to make a fresh start.
```
Comment (Delgan) [inline README.md:88]:
```suggestion
Note that you may also [`remove()`](https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.remove) a previously added handler by using the identifier returned while adding it. This is particularly useful if you want to supersede the default `stderr` handler: just call `logger.remove()` to make a fresh start.
```

**Review by Delgan — CHANGES_REQUESTED** [review]
Good catch!

It must have been inadvertently deleted when I converted the Readme from reStructuredText to Markdown.

The verb is supposed to be "remove", see in 0.7.2 : https://github.com/Delgan/loguru/tree/0.7.2?tab=readme-ov-file

**Review by Delgan — APPROVED** [review]

**Comment by solt87** [comment]
I didn't check any earlier versions, and I (mistakenly...) thought
'call' would also make sense (even though I did read the whole paragraph).
Sorry about that, and thanks for pointing me to the earlier version.

**Comment by Delgan** [comment]
Oh, don't worry, there's nothing to apologize for! It was just for information. :smiley: 

Thanks again for spotting that. :+1:

**Pull request merged**